bitnami/airflow/Chart.yaml

```diff
@@ -32,4 +32,4 @@ name: airflow
 sources:
   - https://github.com/bitnami/bitnami-docker-airflow
   - https://airflow.apache.org/
-version: 8.0.7
+version: 8.0.8
```

Comment:
I thought it was because I considered it a backward-compatible bugfix rather than a new functionality addition. (https://semver.org/).

Deployed releases will only change if they have configured the "KubernetesExecutor", which is the only Executor affected by this configmap.